### DISCUSSION TITLE
Can I refile to a file in a sub directory with the same filename as a file in the root directory?

### DISCUSSION THREAD

**Discussion by linsui** [nvim-orgmode/orgmode#833] [General]
I have some misc.org files in different subdir. It seems I can only refile to the one in the root dir.

**Comment by kristijanhusak** [comment]
How are you setting `org_agenda_files`? It should be a glob pattern to pick up all nested files, like `~/org/**/*`

**Reply by linsui** [reply]
Yes, I set it to `~/Agenda/**/*`. All files are picked except the files with the same name as the one in root dir.

**Reply by kristijanhusak** [reply]
Please give me an example structure, and how are you trying to refile it.

**Reply by kristijanhusak** [reply]
Figured out what's the problem. Opened up issue https://github.com/nvim-orgmode/orgmode/issues/834

**Reply by linsui** [reply]
Thanks!
